migrations/Version202307041134591871_taoTestCenter.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA;
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoTestCenter\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\generis\persistence\PersistenceManager;
+use oat\oatbox\reporting\Report as Report;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+/**
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202307041134591871_taoTestCenter extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Fixes field name for ByPassProctor to a proper one (typo fix).';
+    }
+
+    public function up(Schema $schema): void
```

Comment:
There is one more place where the mistype persists, the value is also stored in the config
https://github.com/oat-sa/extension-tao-testcenter/blob/v10.4.0/config/default/EligibilityCsvImporterFactory.conf.php#L37
Therefore it should be updated as well

Reply:
Could you clarify this? I'm not completely sure what has to happen here, as the place you've mentioned used constant which has already been updated.

Reply:
Fixed. Thank you for your explanation. Every day something new!